x-pack/plugins/security_solution/server/endpoint/errors.ts

```diff
@@ -22,3 +22,10 @@ export class EndpointAppContentServicesNotStartedError extends EndpointError {
     super('EndpointAppContextService has not been started (EndpointAppContextService.start())');
   }
 }
+export class EndpointLicenseError extends EndpointError {
+  public readonly name = 'EndpointLicenseError';
```

Comment:
you don't need this. EndpointError#constructor will set it auto-magically 😄 

see:
https://github.com/elastic/kibana/blob/9fbb61276a901870bf4c3ba42b4795e0d64d689e/x-pack/plugins/security_solution/server/endpoint/errors.ts#L14